### DISCUSSION TITLE
加快Github Pages国内访问速度

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#62] [Comments]
# 加快Github Pages国内访问速度

A boy dreaming of traveling around the world.<br>This blog is used to record my study, thinking and life.

https://zu1k.com/posts/coding/speedup-github-page/

**Comment by lizilong1993** [comment]
大佬博客是hugo+LoveIt吗？感觉好熟悉啊🤭

**Reply by zu1k** [reply]
对，基于 LoveIt 修改了一下

**Reply by hoicau** [reply]
请问可以基于您的博客模板建立自己的博客吗？

**Reply by zu1k** [reply]
可以，随意使用

**Comment by zzzmhcn** [comment]
测试一下

**Comment by MOSSJD** [comment]
博主的博客比我自己的访问快了10倍，看来文章中的坑我都一个不落地踩了。。

**Comment by leostudiooo** [comment]
24 年回复：在某些地区 jsDelivr 已经无法裸连。

**Comment by lmhhh28** [comment]
腾讯云上线了page功能，可以同步github仓库
